bin/sdk_gen

```diff
@@ -172,16 +172,15 @@ const pullci = async (release) => {
 const regen = async (release) => {
   console.info(`Generating SDKs for Looker ${release} ...`)
   if (ok(branchOffMain(release))) {
+    const branch = branchActive()
     if (await pullci(release)) {
       return batch([
         'yarn gen',
         'yarn test:sdk',
-        // 'git add -A',
-        // `git commit -m "feat: generate SDKs for Looker ${release}"`,
-        /*
-        Test these out after first cut is merged to main
-        'git push'
-        create PR
+        'git add -A',
+        `git commit -m "feat: generate SDKs for Looker ${release}"`,
+        `git push origin ${branch}`,
+        /* create PR
```

Comment:
I believe we mentioned this will be removed in a separate PR so I won't let it block this.

Reply:
yeah, I'll get rid of the commented out PR notes for the next round. Thanks!